lib/mustache/parser.rb

```diff
@@ -334,7 +347,13 @@ def scan_tag_delimiter content, fetch, padding, pre_match_position
 
 
     def scan_tag_open_partial content, fetch, padding, pre_match_position
-      @result << [:mustache, :partial, content, offset, padding]
+      @result << if @option_inline_partials_at_compile_time
+        partial = @partial_resolver.call content
+        partial.gsub!(/^/, padding) unless padding.empty?
+        self.class.new(@options).compile partial
+      else
+        [:mustache, :partial, content, offset, padding]
+      end
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [`end` at 346, 6 is not aligned with `if` at 342, 17.](https://www.codacy.com/app/rokusu/mustache/file/2005647324/issues/source?bid=2903613&fileBranchId=3121922#l356)